src/trigger/text_area.cpp

```diff
@@ -45,6 +51,7 @@ TextArea::TextArea(const ReaderMapping& mapping) :
   mapping.get("delay", m_delay);
   mapping.get("once", m_once);
   mapping.get("fade-delay", m_fade_delay);
+  mapping.get("anchor-point", m_anchor, 0);
 
```

Comment:
With this method, the value will appear in the file as a number instead of a string, which isn't very intuitive and memorable for those editing the files manually.

With the New Method:tm:, it will be saved as strings (:tada:), so this line will need to be changed to reading a string and then assigning that string passed to `string_to_anchor_point` to `m_anchor` manually.